types/serverless/plugins/aws/provider/awsProvider.d.ts

```diff
@@ -240,7 +240,7 @@ declare namespace Aws {
     }
 
     interface Logs {
-        restApi?: RestApiLogs;
+        restApi?: RestApiLogs | boolean;
```

Comment:
Is `false` allowed, or should this be `| true`?

Reply:
by the docs, only `true`. Will fix

Reply:
🍏 done